Dockerfile

```diff
@@ -65,9 +65,14 @@ COPY eget_${TARGETARCH:-amd64}.toml /root/.eget.toml
 ARG NODE_EXPORTER_TAG=1.3.1
 # renovate: datasource=github-releases depName=ncabatoff/process-exporter
 ARG PROCESS_EXPORTER_TAG=0.7.10
+# renovate: datasource=github-releases depName=natrontech/openvpn-exporter
+ARG OPENVPN_EXPORTER_TAG=1.0.2
 
 RUN eget prometheus/node_exporter --tag v${NODE_EXPORTER_TAG} \
-	&& eget ncabatoff/process-exporter --tag v${PROCESS_EXPORTER_TAG}
+    && eget ncabatoff/process-exporter --tag v${PROCESS_EXPORTER_TAG} \
+	&& eget natrontech/openvpn-exporter --tag v${OPENVPN_EXPORTER_TAG} \
+		--asset "openvpn-exporter_v${OPENVPN_EXPORTER_TAG}_linux_${TARGETARCH:-amd64}.tar.gz" \
```

Comment:
```suggestion

```

Let's handle this using the eget_(amd64|arm64).toml options.

Reply:
I think I understand why you needed to do this approach -- it's because some packages have filenames that include the filename of the executable's package as a substring.  I found out that `asset_filter` has a way to negate a filter.  We can use the following option in the eget toml configuration instead of filtering it during build time:
```
asset_filters = [".tar.gz", "^.sbom.json"]
```